src/test/java/reposense/authorship/FileInfoExtractorTest.java

```diff
@@ -68,6 +68,23 @@ public void extractFileInfosTest() {
         Assertions.assertTrue(isFileExistence(Paths.get("newFile.java"), files));
     }
 
+    @Test
+    public void extractFileInfos_skipGlobs_success() {
+        config.addAuthorNamesToAuthorMapEntry(new Author(MAIN_AUTHOR_NAME), MAIN_AUTHOR_NAME);
+        config.addAuthorNamesToAuthorMapEntry(new Author(FAKE_AUTHOR_NAME), FAKE_AUTHOR_NAME);
+        config.setIgnoreGlobList(Arrays.asList("newPos/**.java", "**.md"));
+
+        GitCheckout.checkout(config.getRepoRoot(), TEST_COMMIT_HASH);
+        List<FileInfo> files = FileInfoExtractor.extractTextFileInfos(config);
+
+        Assertions.assertEquals(4, files.size());
+
+        Assertions.assertTrue(isFileExistence(Paths.get("annotationTest.java"), files));
+        Assertions.assertTrue(isFileExistence(Paths.get("blameTest.java"), files));
+        Assertions.assertTrue(isFileExistence(Paths.get("inMasterBranch.java"), files));
+        Assertions.assertTrue(isFileExistence(Paths.get("newFile.java"), files));
+    }
```

Comment:
I think this is missing the check for files that are correctly ignored, such as some `.md` files or `newPos/*.java` files.